src/storage/client/StorageClient.cpp

```diff
@@ -94,10 +94,10 @@ folly::SemiFuture<StorageRpcResponse<cpp2::ExecResponse>> StorageClient::addEdge
 folly::SemiFuture<StorageRpcResponse<cpp2::QueryResponse>> StorageClient::getNeighbors(
         GraphSpaceID space,
         std::vector<VertexID> vertices,
-        EdgeType edgeType,
-        bool isOutBound,
+        std::vector<EdgeType> edgeTypes,
```

Comment:
the caller in GoExecutor::stepOut is pass parameter by value, so here maybe const std::vector<EdgeType> &edgeTypes is better, or req.set_edge_types(std::move(edgeTypes));